bundle/Resources/views/ContentType/field_types.html.twig

```diff
@@ -116,6 +116,27 @@
     </div>
 {% endblock %}
 
+{% block ezobjectrelationlist_field_definition_edit %}
+    <div class="ezobjectrelationlist-settings selection-default-location">
+        {{- form_label(form.selectionDefaultLocation) -}}
+        {{- form_errors(form.selectionDefaultLocation) -}}
+        {{- form_widget(form.selectionDefaultLocation) -}}
+
+        <button data-universaldiscovery-title="{{"field_definition.ezobjectrelationlist.selection_root_udw_title"|trans({}, "ezrepoforms_content_type")}}"
+                class="ez-button-tree pure-button ez-font-icon ez-button ez-relation-pick-root-button"
+                data-relation-root-input-selector="#{{form.selectionDefaultLocation.vars.id}}"
+                data-relation-selected-root-name-selector="#selected_item_name"
+                >{{"field_definition.ezobjectrelationlist.selection_root_udw_button"|trans({}, "ezrepoforms_content_type")}}</button>
+        <div id="selected_item_name"></div>
```

Comment:
the `id` won't be unique in the page if there are several relation list field definitions in a content type so it should contain the field definition identifier or something like that. In addition, you should use `-` as a separator instead of `_` and we can probably find a better name like `relation-selected-root-name` to be consistent with the button data attribute. And last, this should be pre-filled with the name of content if the field definition already has a selected root.